buildAndReleaseTask/tests/_suite.ts

```diff
@@ -3,50 +3,67 @@
 import * as assert from "assert";
 import * as ttm from "azure-pipelines-task-lib/mock-test";
 import * as path from "path";
-import { userRequestedVersion } from "./success";
+
+function mustNotHaveErrorsOrWarnings(tr: ttm.MockTestRunner) {
```

Comment:
Other than adding the new test case to this test suite, I did some minor cleanup.